okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -241,15 +240,7 @@ public Level getLevel() {
         Charset charset = UTF8;
         MediaType contentType = responseBody.contentType();
         if (contentType != null) {
-          try {
```

Comment:
I don't have a strong opinion. A reasonable fallback seems fine to me. @dave-r12 added the original code here. Doesn't look like was motivated by a particular issue.